include/structures/junction.hpp

```diff
@@ -29,8 +30,10 @@ class Junction
     Junction(Breakend the_mate1,
              Breakend the_mate2,
              auto const & the_inserted_sequence,
+             int16_t const & the_tandem_dup_count,
```

Comment:
No advantage of using `const&`
```suggestion
             int16_t the_tandem_dup_count,
```